LICENSE

```diff
@@ -1,685 +1,29 @@
-All code within the microstrain_3dm_gx5_45 package that was developed by Brian Bingham is released under the GPL license:
-
-Copyright (c) 2016, Brian Bingham
-All rights reserved.
+All code within the microstrain_3dm_gx5_45 package that was developed by Brian Bingham is released under the MIT license.
```

Comment:
Should `microstrain_3dm_gx5_45` become `microstrain_mips`?

Reply:
Yes - good catch!